internal/provider/webservice/models.go

```diff
@@ -79,7 +79,7 @@ func ModelForServiceResult(service *common.WrappedService, plan WebServiceModel,
 		LogStreamOverride:    common.LogStreamOverrideFromClient(service.LogStreamOverride, plan.LogStreamOverride, diags),
 	}
 
-	runtimeSource, err := common.RuntimeSourceFromClient(service.Service, details.Env, details.EnvSpecificDetails)
+	runtimeSource, err := common.RuntimeSourceFromClient(service.Service, details.Runtime, details.EnvSpecificDetails)
```

Comment:
I'm pretty sure the answer is yes but do `Env` and `Runtime` have the same values? Just want to make sure this won't change tf state

Reply:
Yeah they represent the same value. We just updated the field name.